index.html

```diff
@@ -4,9 +4,78 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="styles/index.css" rel="stylesheet">
+    <!-- <script src="./src/index.js"></script> -->
+    <script src="./node_modules/axios/dist/axios.min.js"></script>
```

Comment:
Looks like you have two script tags that brings loads axios. This same element is also on line 78.

Be sure to remove this script at the beginning of the page since "When the browser encounters a <script> tag, it stops loading the HTML document. Instead, the browser pauses to download the entire script, which might take a long time to load. The browser continues rendering the page only after the script has finished downloading." (from our Learn reading)

https://learn-2.galvanize.com/cohorts/3169/blocks/1449/content_files/adding-behavior/connecting-html-css-and-js.md